test/integration/volume_integration_test.go

```diff
@@ -186,20 +186,3 @@ func createVolumeInZoneAndAssert(t *testing.T, zone cloudscale.Zone, wg *sync.Wa
 		t.Errorf("Volumes.Delete returned error %s\n", err)
 	}
 }
-
-func TestIntegrationVolume_DeleteRemainingVolumes(t *testing.T) {
-	volumes, err := client.Volumes.List(context.Background(), nil)
-	if err != nil {
-		t.Fatalf("Volumes.List returned error %s\n", err)
-	}
-
-	for _, volume := range volumes {
-		if strings.HasPrefix(volume.Name, "go-sdk-integration-test") {
-			t.Errorf("Found not deleted volume: %s\n", volume.Name)
-			err = client.Volumes.Delete(context.Background(), volume.UUID)
-			if err != nil {
-				t.Errorf("Volumes.Delete returned error %s\n", err)
-			}
-		}
-	}
-}
```

Comment:
I personally think that this test (and the one above about volumes makes sense in a way. It shows that previous runs haven't cleaned up properly. What do you think?

Reply:
I agree with you. The problem was that it is not sufficient to just check for volumes in the volumes test, there are servers as well. The same applies for Networks, Floating IPs, etc. It became kind of a mess.

This is why I have moved all the `DeleteRemaining*` stuff to the `cloudscale_test.go` file. It will still fail the test run if there are remaining resources. Btw my idea is to use tags for this purpose once we have implemented them for all resources in the go-sdk. 

See:
https://github.com/cloudscale-ch/cloudscale-go-sdk/blob/6c27a9a5de2a7405fa5cd51e6d29bd63a3c62209/test/integration/cloudscale_test.go

What do you think?

Reply:
That sounds good!